packages/now-routing-utils/test/index.spec.js

```diff
@@ -54,6 +54,10 @@ describe('normalizeRoutes', () => {
       },
       { handle: 'filesystem' },
       { src: '^/(?<slug>[^/]+)$', dest: 'blog?slug=$slug' },
+      { handle: 'hit' },
+      { src: '^/hit-me$', dest: '/api/hit-miss' },
+      { handle: 'miss' },
+      { src: '^/missed-me$', dest: '/api/missed-me' },
```

Comment:
Routes after `handle: miss` must set `check: true`, or if there's no `dest` `continue: true` instead

Reply:
Thanks, I updated the logic 👍